apps/storybook/src/TeamsTabbedCard.stories.tsx

```diff
@@ -9,7 +9,13 @@ export default {
 
 export const Normal = () => (
   <TeamsTabbedCard
-    teams={createListTeamResponse(number('Teams', 10)).items}
+    teams={createListTeamResponse(number('Teams', 10)).items.map(
+      (team, index) => ({
+        ...team,
+        inactiveSince: index === 2 ? '2021-01-01' : undefined,
+      }),
+    )}
     title={text('Title', 'Interest Group Teams')}
+    isGroupInactive={boolean('Is Group Inactive?', false)}
```

Comment:
I don't like this prop. This is a Teams Tabbed Card, either it needs to be a GroupTeamsTabbedCard or isGroupInactive should be something more generic